injector/ProgramProcessor.py

```
@@ -1,10 +1,10 @@
-from injector import helper
-import pathlib
 import shutil
-import json
-import ast
 import os
-from injector.LogInjector import LogInjector
+import ast
+from pathlib import Path
+from injector import helper
+from injector.FindLocalImports import findLocalImports
+from injector.LogInjectorVisitor import LogInjectorVisitor
```

Ruff (0.8.2)

`injector.LogInjectorVisitor.LogInjectorVisitor` imported but unused

Remove unused import: `injector.LogInjectorVisitor.LogInjectorVisitor`

(F401)